### PR TITLE
WIP: Add kubectl events command

### PR DESCRIPTION
**Pull request by soltysh**
This PR documents the alpha feature kubernetes/enhancements#1440, `kubectl events`.

### CODE REVIEW

**Comment by kbhawkey** [comment]
Hello @soltysh , If this change is for the 1.18 release, you should use the dev-1.18 branch.

**Comment by sftim** [comment]
/hold

because this PR seems to target the wrong branch

File: content/en/docs/reference/kubectl/overview.md [hunk 1]
```diff
@@ -81,6 +81,7 @@ Operation       | Syntax    |       Description
 `describe`    | `kubectl describe (-f FILENAME | TYPE [NAME_PREFIX | /NAME | -l label]) [flags]` | Display the detailed state of one or more resources.
 `diff`        | `kubectl diff -f FILENAME [flags]`| Diff file or stdin against live configuration (**BETA**)
 `edit`        | `kubectl edit (-f FILENAME | TYPE NAME | TYPE/NAME) [flags]` | Edit and update the definition of one or more resources on the server by using the default editor.
+`events`      | `kubectl events | List events.
```
Comment (sftim) [inline content/en/docs/reference/kubectl/overview.md]:
```suggestion
`events`      | `kubectl events | List events (**ALPHA**).
```
?

Do users have to do anything to make alpha subcommands work?

**Comment by jimangel** [comment]
@soltysh please target this PR to dev-1.18 - thanks!

/cc @VineethReddy02

**Comment by sftim** [comment]
@soltysh we (SIG Docs) haven't heard from you in a while.

If you want to propose a similar change for a future release, please go ahead.

/close

**Pull request closed without merging**